.gitignore

```diff
@@ -44,3 +44,4 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+/src/environments/**
```

Comment:
I meant "Environment files added to git .ignore" and not "Environment files added to git commits" as mentioned in the commit message